modules/Dial_Display.md

```diff
@@ -0,0 +1,65 @@
+Bangle.js Dial Display Library
+======================
+
+
+> Take a look at README.md for hints on developing with this library.
+
+Usage
+-----
+
+```JS
+var DialDisplay = require("Dial_Display");
+var dialDisplay = new DialDisplay(options);
```

Comment:
Same again, but here I think we'll want to change to make it more class-like, or alter how `this` is used within the `dialDisplay` function. At the moment it'll be writing to the global object.

For example, in the IDE:
```javascript
> cb
=function (step) { ... }
>cb(1)
=undefined
>cb(1)
=undefined

// the dial renders successfully
// `this` variables are written to the global:

>isFirstDraw
=false
>value
=14
```

Happy to go through a few suggestions for it, if you like

Reply:
Of course, I'll pop something together

Reply:
[Here you go](https://github.com/espruino/BangleApps/pull/3847/commits/91a6b9d78638c7ea3f7737c877a18aa7c144d4b6?diff=split&w=1) - I've summarised the usage in the updated readme (inline below), let me know what you think. Happy to revert if you'd prefer a different approach :)


https://github.com/espruino/BangleApps/blob/91a6b9d78638c7ea3f7737c877a18aa7c144d4b6/modules/Dial_Display.md#L11-L20

Reply:
Yes very true, `reset` is a little ambiguous! (feel free to change it, delete any of the methods or anything of course)

But yes - I found exactly what you said, `set()` makes it forget the previous value, so I altered the code slightly to deal with that too

Reply:
> so I altered the code slightly to deal with that too

I don't think you pushed this? 🙂

Reply:
Ah, sorry I should've said - it was all in the same commit, the code keeping `prevValue` in sync is here :)

https://github.com/espruino/BangleApps/blob/91a6b9d78638c7ea3f7737c877a18aa7c144d4b6/modules/Dial_Display.js#L24-L32